Gemfile

```diff
@@ -45,6 +45,8 @@ gem "bootsnap", require: false
 # Use Sass to process CSS
 gem "sassc-rails"
 
```

Comment:
Suggest to add comment "# Custom gems" and moving forward we could put all additional gems under this comment